src/plugins/dashboard_embeddable_container/public/embeddable/dashboard_container.tsx

```diff
@@ -53,6 +53,7 @@ export interface DashboardContainerInput extends ContainerInput {
   useMargins: boolean;
   title: string;
   description?: string;
+  isEmbedded: boolean;
```

Comment:
An embeddable is always embedded within some other container.
In the context of this PR, what does `isEmbedded` mean?
We might need a different terminiology.